x/staking/types/exported.go

```diff
@@ -11,7 +11,7 @@ import (
 
 // DelegationI delegation bond for a delegated proof of stake system
 type DelegationI interface {
-	GetDelegatorAddr() sdk.AccAddress // delegator sdk.AccAddress for the bond
+	GetDelegatorAddr() string         // delegator string for the bond
 	GetValidatorAddr() sdk.ValAddress // validator operator address
```

Comment:
i can migrate this one once we merge https://github.com/cosmos/cosmos-sdk/pull/16959, that way this interface is consistent

Reply:
@tac0turtle why not use bytes?

Reply:
would need to do a state migration, this bled into state, we will get rid of it in the next release or the upcoming rewrite of staking